cmd/atlas/internal/client/doc.go

```diff
@@ -0,0 +1,7 @@
+// Package client provides abstraction over the usage of Atlas.
+// Example usage of client:
+//		d,close,_ := client.NewAtlasDriver(dsn)
+//		defer close()
+//		data := client.Inspect(ctx, d)
+//
+package client
```

Comment:
`cmd` is not importable. Can we start with developing stuff on `cmd/atlas` and move them to a shared place when it's necessary? I also think that the package name `client` is too generic (like `util`) and should not exist as a namespace.